.gitignore

```diff
@@ -11,6 +11,6 @@ plugins/
 *.swo
 *.log
 *.DS_Store
-www/ionic.app.css
-www/ionic.app.min.css
+www/css/ionic.app.css
+www/css/ionic.app.min.css
```

Comment:
don't add this, please remove them and commit. Those styles are generated, it's already added to gitignore.